tests/foreman/cli/test_contentview.py

```diff
@@ -5136,7 +5139,49 @@ class ContentViewFileRepoTestCase(CLITestCase):
     """Specific tests for Content Views with File Repositories containing
     arbitrary files
     """
-    @stubbed()
+    @classmethod
+    def setUpClass(cls):
+        """Create a product and an org which can be re-used in tests."""
+        super(ContentViewFileRepoTestCase, cls).setUpClass()
+        cls.org = make_org()
+        cls.product = make_product({'organization-id': cls.org['id']})
+
+    def _make_file_repository_upload_contents(self, options=None):
+        """Makes a new File repository, Upload File/Multiple Files
+        and asserts its success """
+        if options is None:
+            options = {}
+
+        options_list = (u'product-id', u'content-type', u'url', u'name')
+        if None in [options.get(keys) for keys in options_list]:
+            options[u'product-id'] = self.product['id']
+            options[u'content-type'] = 'file'
+            options[u'url'] = CUSTOM_FILE_REPO
+            options[u'name'] = gen_string('alpha')
+
+        new_repo = make_repository(options)
+        remote_path = "/tmp/{0}".format(RPM_TO_UPLOAD)
```

Comment:
`os.path.join('/tmp', RPM_TO_UPLOAD)` is cleaner

Reply:
@ekohl we cannot use os.path.join for remote path as it's intended for local os script run time and here we are creating a remote path, suppose that you run os.path.join under windows what the path will be?

Reply:
I'm not sure `/tmp` works on Windows, but you're right that it won't work for remote paths in mixed environments.